_scripts/instruction-widget/install.js

```diff
@@ -95,6 +97,7 @@ module.exports = function(context) {
 
   debian_install = function() {
     template = "debian";
+    context.devuan = context.distro == "devuan"
```

Comment:
`context.devuan` is set to true only if `context.distro` is "devuan"`. If it is, we'll use the Devuan instructions, otherwise, we'll use the Debian instructions.